### DISCUSSION TITLE
アーティファクト生成の巻物の強化要望

### DISCUSSION THREAD

**Discussion by Aquashrimp** [hengband/hengband#218] [Ideas]
勝手版からの逆輸入になりますが、生成の巻物を指輪やアミュにも使えるようにして頂きたいです

**Comment by deskull-m** [comment]
正直、今の強化具合が勝手版と比べても貧弱すぎる観点から、それ出来てもいいかもしれませんね。議題とします。

**Reply by iks3** [reply]
鎧なんかは全耐性(4耐性)持ちもしくはそれ相当の耐性持ちの装備ができやすくてもいいかもしれませんね。
